frontend/src/lib/components/Top.svelte

```diff
@@ -57,16 +58,23 @@
         border: 0;
     }
 
-    #logout::before{
+    button::before{
```

Comment:
I would use class for this
Why?
If someone will try to expand this component, he might get confused why button has some weird stuff.

Because of that I recommend using element css selectors only with elements like `main` or `html` which usually don't occur more than once in a component.